src/metabase/sync/sync_metadata/tables.clj

```diff
@@ -80,6 +81,15 @@
 
 ;;; ---------------------------------------------------- Syncing -----------------------------------------------------
 
+(s/defn ^:private update-database-metadata!
+  "If there is a version in the db-metadata update the DB to have that in the DB model"
+  [database :- i/DatabaseInstance db-metadata :- i/DatabaseMetadata]
+  (log/info (trs "Found new version for DB:")
```

Comment:
Does this add a space between the message and the param value?  I haven't seen this usage of `trs` before (only seen using, like `{0}` and then making the val appear as an additional param after the message).